installers/Windows/README.md

```diff
@@ -16,10 +16,19 @@ Script to build an installer for Windows using Pynsist.
 Mu is distributed under the GPL-3.0 license.
 
 
-Use the base functions of the script to build a Windows installer of spyder,
-mainly changing the code to generalize its functionality and configure the run
-with more arguments (package name, conda python interpreter, extra packages,
-etc.).
+We use here the base functions of the script to build a Windows installer of
+spyder, mainly changing the code to generalize its functionality and configure
+the run with more arguments (package name, conda python interpreter, extra
+packages, etc.).
+
+As a NSIS installer the generated installer counts with some command line flags:
+https://nsis.sourceforge.io/Docs/Chapter3.html#installerusage
+
+Besides those flags, the installer also counts with the command line options
+available when using the `MultiUser.nsh` header file:
```

Comment:
```suggestion
Besides those flags, the installer also has with the following command line options
available because it uses the `MultiUser.nsh` header file:
```

I'm not so sure about this suggestion, but that's what I understand from what you wrote.